app/constants/AppConstants.js

```diff
@@ -68,4 +68,10 @@ export default {
   TIME_FORMAT: 'H:mm',
   TIME_SLOT_DEFAULT_LENGTH: 30,
   TRACKING: SETTINGS.TRACKING,
+  SORT_BY_OPTIONS: {
+    RESOURCE_NAME: 'resource_name_lang',
+    TYPE: 'type_name_lang',
+    PREMISES: 'unit_name_lang',
+    PEOPLE: 'people_capacity',
```

Comment:
put a TODO: add sortby `openingNow` feature comment for later development

Reply:
Done!